model/simulationtests/flex_measures_dual_source.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+require 'openstudio'
+require 'lib/baseline_model'
+
+model = BaselineModel.new
+
+# make a 1 story, 100m X 50m, 10 zone core/perimeter building
+model.add_geometry({ 'length' => 100,
+                     'width' => 50,
+                     'num_floors' => 1,
+                     'floor_to_floor_height' => 4,
+                     'plenum_height' => 1,
+                     'perimeter_zone_depth' => 3 })
```

Comment:
same comment for all files. This is currently a 5 zones + 1 plenum building. Simplify if you can, our overall test runtime will thank you! :)